File: src/main/java/it/tylframework/data/mongo/basics/NumeratorType.java
package it.tylframework.data.mongo.basics;

import it.tylframework.data.mongo.common.Footprint;
import it.tylframework.data.mongo.common.MlText;
import lombok.AllArgsConstructor;
import lombok.Data;
import org.springframework.data.annotation.TypeAlias;

/**
 * Created by mp on 20/11/14.
 */
@TypeAlias("bas_NumeratorType")
@Data
@AllArgsConstructor
public class NumeratorType extends Footprint {
    MlText name;
    MlText description;
    Boolean ordered_by;
    Boolean automatic_attribution;
    Boolean manual_admitted;
}

File: src/main/java/it/tylframework/data/mongo/basics/NumeratorType.kt
package it.tylframework.data.mongo.basics

import it.tylframework.data.mongo.common.MlText
import it.tylframework.data.mongo.common.Footprint
import it.tylframework.data.mongo.config.TylContext
import java.util.Date

/**
 * Created by marco on 19/09/14.
 */

data class NumeratorType(
        var name:MlText,
        var description:MlText,
        var orderedBy:Boolean,
        var automaticAttribution:Boolean,
        var manualAdmitted:Boolean)
        :Footprint(TylContext.currentUser, Date() ,TylContext.currentUser,  Date()
){}